### DISCUSSION TITLE
VCA GPS Konturen Fahren

### DISCUSSION THREAD

**Discussion by Nightlife121** [Mogli12/KeyboardSteer#860] [Ideas]
Guten Tag, 
als gelernter Landwirt auf großen Maschinen mit GPS kam mir so mancher Gedanke und Funktionen die man vielleicht auch im spiel umsetzten könnte. 

- Konturen erstellen um Kurvige Felder auch im Vorgewende mit GPS fahren zu können und
- das speichern von Bahnen wäre in diesen Zuge auch nicht schlecht da man sonst alles wieder neu einfahren müsste.

**Comment by Mogli12** [comment]
Hallo,

die Idee finde ich zwar sehr gut, da ich nur noch wenig mit Helfer spiele. Aber das wird etwas dauern. 

Schwierig ist vor allem, dass man den Kurs für die Rückrichtung um den doppelten Abstand zwischen Traktor und Gerät verschieben muss.

Viele Grüße
